### PR TITLE
OKTA-496101 [fixit] myaccount api developer documentation fix.

### PR DESCRIPTION
**Pull request by JayHe-okta**
## Description:
- **What's changed?** 

1. address the remaining comments on the my account developer doc for profile endpoints.
2. add error response across the documentation to make the endpoints consistent
3. some minor wording fixes. 
- **Is this PR related to a Monolith release?** 
N/A

### Resolves:

* [OKTA-496101](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 1]
```diff
@@ -254,6 +254,14 @@ This API requires a [My Email Request object](#my-email-request-object) as its r
 
 The requested [My Email object](#my-email-object)
 
+#### Error Responses
+
+If an invalid email is passed to `profile` in the request body, the response will return a 400 BAD REQUEST with error code E0000001.
+
+If the email operation is not enabled for the request `role` on the org, the response will return a 403 FORBIDDEN with error code E0000038.
```
Comment (hanyuanzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:261]: Is there a clearer way of saying "email operation"?

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 2]
```diff
@@ -419,6 +433,16 @@ This request requires the `verificationCode` property as its request body.
 
 #### Response body
 
+N/A
+
+#### Error Responses
+
+if the requested `verificationCode` is invalid, the response will return a 401 UNAUTHORIZED with error code E0000004.
```
Comment (hanyuanzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:440]: `if` -> `If`

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 3]
```diff
@@ -1073,14 +1133,18 @@ This API requires the `profile` property of a [User Profile](#user-profile-objec
 
 Returns the result of applying the update, as if the caller had invoked the GET User Profile operation.
 
+#### Error Responses
+
+if provided profile attributes for update is invalid, the response will return a 400 BAD REQUEST with error code E0000001.
```
Comment (hanyuanzhang-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:1138]: `if` -> `If`

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 4]
```diff
@@ -254,6 +254,14 @@ This API requires a [My Email Request object](#my-email-request-object) as its r
 
 The requested [My Email object](#my-email-object)
 
+#### Error Responses
+
+If an invalid email is passed to `profile` in the request body, the response will return a 400 BAD REQUEST with error code E0000001.
```
[the remark this answers is not in the document]
Comment (JayHe-okta) [reply packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:259]: yeah but this is not for profile endpoints tho. this is for the myaccount email endpoints.

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md (hunk 1 above)
Comment (JayHe-okta) [reply packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:261]: discussed with Hanyuan offline email operation probably ok..

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (justinabrokwah-okta) [reply packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:259]: Ah my mistake, yeah just realized I misread this. Sorry, this looks good!

File: packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md [hunk 5]
```diff
@@ -521,6 +546,12 @@ N/A
 
 N/A
 
+#### Error Responses
+
+If an invalid emailId is requested for deletion, the response will return a 404 NOT FOUND with error code E0000007.
+
+If the email for deletion is an already verified email, the response will return a 400 BAD REQUEST with error code E0000001.
```
Comment (ericalong-okta) [inline packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md:553]: nit: the ordering

**Review by ericalong-okta — APPROVED** [review]

**Review by justinabrokwah-okta — APPROVED** [review]
LGTM

**Pull request merged**